tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -442,11 +442,9 @@ void handleRequestImpl(sourcekitd_object_t ReqObj, ResponseReceiver Rec) {
     if (!Req.getInt64(KeyOptimizeForIDE, EditorMode, true)) {
       OptimizeForIDE = EditorMode;
     }
-    Optional<unsigned> CompletionCheckDependencyInterval;
-    int64_t IntervalValue = 0;
-    if (!Req.getInt64(KeyCompletionCheckDependencyInterval,
-                      IntervalValue, /*isOptional=*/true))
-      CompletionCheckDependencyInterval = IntervalValue;
+    Optional<unsigned> CompletionCheckDependencyInterval =
+      Req.getOptionalInt64(KeyCompletionCheckDependencyInterval)
+        .map([](int64_t v)->unsigned{return v;});
```

Comment:
Just to clarify, should rewriting ``/*isOptional=*/true` to `/*isOptional=*/false` in the original code also fix the issue?

Reply:
Yes, but `getOptionalInt64()` is more readable. I have to admit that `getInt64()` is not well thought out as an API and rife for confusion, as in this case.

Reply:
Thanks! I agree `getOptionalInt64()` is more readable.